specify v2 api as an option as well (#9096)
circleci/circleci-docs: jekyll/_cci2/github-apps-integration.adoc

[#scheduled-pipelines]
=== Scheduled pipelines

> The ability to xref:scheduled-pipelines#[schedule pipelines] is not currently supported for GitHub App projects. This feature is planned for a future release. One alternative is to use a xref:webhooks/#custom-webhooks[custom webhook] or the xref:triggers-overview/#run-a-pipeline-using-the-api[V2 API to trigger a pipeline] that you `curl` with a 3rd party scheduling tool.

Reviewer: ```suggestion
The ability to xref:scheduled-pipelines#[schedule pipelines] is not currently supported for GitHub App projects. This feature is planned for a future release. As an alternative, use a xref:webhooks/#custom-webhooks[custom webhook] or the xref:triggers-overview/#run-a-pipeline-using-the-api[V2 API to trigger a pipeline], which you can `curl` with a 3rd party scheduling tool.
```
